### ISSUE TITLE
Support alt text for images added through Appearance Menu

### ISSUE THREAD

**Issue by laritakr** [samvera/hyku#1812]
### Descriptive summary

Contribute back Accessibility changes

Add the ability to add alt text for each of images added through Accessibility menu.

The following home page should have an alt text, if one is set.
* [ ] Logos
* [ ] Banner Image

### Rationale

Accessibility issue remediation

### Steps to reproduce the behavior

1. Do this
1. Then do this...

### Related work

Link to related tickets or prior related work here.
https://gitlab.com/notch8/utk-hyku/-/issues/54

**Comment by posixeleni** [comment]
Hi @laritakr this has been resolved with at least 7.1 right?

**Comment by laritakr** [comment]
> Hi [@laritakr](https://github.com/laritakr) this has been resolved with at least 7.1 right?

The appearance menu items were resolved long ago.
